WordPressKit/PostServiceRemoteREST.m

```diff
@@ -432,7 +436,21 @@ - (RemotePost *)remotePostFromJSONDictionary:(NSDictionary *)jsonPost {
     post.tags = [self tagNamesFromJSONDictionary:jsonPost[@"tags"]];
 
     post.revisions = [jsonPost arrayForKey:@"revisions"];
-    
+
+    NSDictionary *autosaveAttributes = jsonPost[@"meta"][@"data"][@"autosave"];
+    if (autosaveAttributes) {
```

Comment:
If `autosaveAttributes` is `null` (in the response JSON), it will be deserialized into `NSNull` in the dictionary - only if the key is not present will it return `nil`. I'm going to update this to `wp_isValidObject` to handle this case.

https://github.com/wordpress-mobile/WordPressKit-iOS/blob/78a0a0ef53e7a5c5890930cd876328c3688e467c/WordPressKit/ObjectValidation.swift#L9

Reply:
Great find! Thanks for fixing it.